public/css/style.css

```diff
@@ -47,3 +47,31 @@ a.navbar-brand {
 select#languages {
     height: 30em;
 }
+
+h2 {
```

Comment:
These (h2 and h3) are messing with headers everywhere. Please remove these.

For reference, this is what is used on the site:
https://getbootstrap.com/docs/3.3/css/
https://getbootstrap.com/docs/3.3/components/

Reply:
OK. I'd have hoped that this would show the "Add project" etc. stuff not so very huge as it is now...

Reply:
OK. I'd have hoped that this would show the "Add project" etc. stuff not so very huge as it is now...